intro-HPC/ch_troubleshooting.tex

```diff
@@ -200,3 +200,48 @@ \section{Module conflicts}
 that have the same version of common dependencies. You can list all versions of
 a module with \verb|module avail|: for \verb|HMMER|, this command is \verb|module avail HMMER|.
 \fi
+
+\ifgent
+
+\section{Running software that is incompatible with host}
+
+\begin{prompt}
+%\shellcmd{module swap cluster/golett}%
+
+The following have been reloaded with a version change:
+  1) cluster/delcatty => cluster/golett
+
+%\shellcmd{module load Python/2.7.14-intel-2018a}%
+%\shellcmd{python}%
+
+Please verify that both the operating system and the processor support Intel(R) MOVBE, F16C, FMA, BMI, LZCNT and AVX2 instructions.
+
+\end{prompt}
```

Comment:
or:

```
Illegal instruction
```

(you'll get that with the same example but using the `Python/2.7.14-foss-2018a` module for example)